docs/local-development.md

```diff
@@ -580,27 +580,31 @@ index a3b2ba6..cd5a8e7 100644
 
 ### Linting
 
-| Type     | Command              |
-| -------- | -------------------- |
-| all      | `yarn run lint`      |
-| css      | `yarn run lint:jest` |
-| js       | `yarn run lint:jest` |
-| markdown | `yarn run lint-md`   |
+| Type     | Command             |
+| -------- | ------------------- |
+| all      | `yarn run lint`     |
+| css      | `yarn run lint:css` |
```

Comment:
We swapped back and forth on using `-` and `:` in the config so I just settled on `:`